tests/integration/conftest.py

```diff
@@ -173,10 +181,10 @@ def fixture_zkserver(session_tmppath: Path) -> Iterator[Optional[ZKConfig]]:
     # transfer_file (primarily the server's port number)
     with FileLock(str(lock_path_for(zk_dir))):
         if transfer_file.exists():
-            config = pickle.loads(transfer_file.read_bytes())
```

Comment:
this was breaking when running the tests with `pytest tests`, it only worked if `pytests tests/integration` was used because of how `pytest` messes with the import paths.